tests/gbs/test_vibronic.py

```diff
@@ -67,6 +76,16 @@ def test_duschinsky(self):
         Ud = np.diag(self.wp ** -0.5) @ J @ np.diag(self.w ** 0.5)
         assert np.allclose(Ud, self.Ud)
 
+    def test_invalid_temperature(self):
+        """Test if function raises a ``ValueError`` when a negative temperature is given."""
+        with pytest.raises(ValueError, match="Temperature must be zero or positive"):
+            vibronic.gbs_params(self.w, self.wp, self.Ud, self.d, -1)
+
+    def test_twomode(self):
+        """Test if function returns two-mode squeezing parameters that correctly reconstruct the
+        input normal mode frequencies."""
+        w = -0.02 * self.k * self.T / self.h / self.c * np.log(np.tanh(self.t))
```

Comment:
```suggestion
        w = - self.k * self.T / (0.5 * h * w * c * 100) * np.log(np.tanh(self.t))
```
Just to make it look more like the inverse of the function, e.g., the 0.02 confused me at first